context/build.gradle

```diff
@@ -2,5 +2,6 @@ description = 'gRPC: Context'
 
 dependencies {
     testCompile project(':grpc-testing')
+    testCompile project(':grpc-core').sourceSets.test.output
```

Comment:
Maybe move StaticTestingClassLoader to context and have core depend on context's test?